packages/dcos-history/build

```diff
@@ -3,12 +3,18 @@ source /opt/mesosphere/environment.export
 export LIB_INSTALL_DIR="$PKG_PATH/lib/python3.4/site-packages"
 mkdir -p "$LIB_INSTALL_DIR"
 
+
 # Copy from the ro /pkg/extra to a rw folder.
 cp -r /pkg/extra /build
 
+
+# Copy the gunicorn conf file to $PKG_PATH
+envsubst '$PKG_PATH' < /pkg/extra/history/dcos_history_conf.py > "$PKG_PATH/dcos_history_conf.py"
+
+
 pip3 install --no-deps --install-option="--prefix=$PKG_PATH" --root=/ /build
 
 # Create the service file
 service="$PKG_PATH/dcos.target.wants_master/dcos-history.service"
 mkdir -p "$(dirname "$service")"
-cp /pkg/extra/dcos-history.service  "$service"
+envsubst '$PKG_PATH' < /pkg/extra/dcos-history.service > "$service"
```

Comment:
the service has a $PKG_PATH in it. https://github.com/dcos/dcos/pull/749/files#diff-ee8c631a232fb776584c0b43f493138eR15